examples/report_generation/rfp_response/generate_rfp.ipynb

```diff
@@ -0,0 +1,1096 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #155.                    docs = [Document.parse_obj(json.loads(line)) for line in f]
</pre><p>Not: <code>parse_obj</code> is deprecated in pydantic, should be <code>docs = [Document.model_validate_json(line) for line in f]</code>  I think</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/438/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>